File: year2020/month09/day0918/gebal_0918.py
def compare(t):
    a, b = list(map(t, input().split(' ')))
    is_true = abs(a) > abs(b)
    is_true = not is_true if t == float else is_true
    return a if is_true else b

print(compare(int))
print(compare(float))

File: year2020/month09/day0918/gebal_0918.c
#include<stdio.h>
#include<math.h>

void compare(int type, void *a, void *b) {
	if (type == sizeof(int)) {
		int x = *(int*)a;
		int y = *(int*)b;
		printf("%d\n", abs(x) > abs(y) ? x : y);
	}
	else {
		double x = *(double*)a;
		double y = *(double*)b;
		printf("%0.2lf\n", (fabs(x) > fabs(y) ? y : x));
	}
}

int main() {
	int a, b;
	double ra, rb;
	scanf("%d %d", &a, &b);
	compare(sizeof(a), &a, &b);	
	scanf("%lf %lf", &ra, &rb);
	compare(sizeof(ra), &ra, &rb);
}
